MvvmCross/ViewModels/MvxNavigationViewModel.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MS-PL license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using MvvmCross.Logging;
+using MvvmCross.Navigation;
+
+namespace MvvmCross.ViewModels
+{
+    public abstract class MvxNavigationViewModel
```

Comment:
Shouldn't we name it differently? `MvxNavigationViewModel` sounds like it is only for Navigation. What about `MvxBaseViewModel` or something?

Reply:
MvxBaseViewModel sounds like it should be the base class of MvxViewModel, not the other way around. I think there are typically two types of viewmodels that developers create - those that are used as part of navigation (ie would inherit from MvxNavigationViewModel) and those that are used to populate lists (or other non-navigation related scenarios, and would inherit from MvxViewModel). This is the reason I set the name as MvxNavigationViewModel 

Having said that, happy to consider other names. The other alternative is that we switch the names - this would be MvxViewModel, and the class that is currently MvxViewModel would become MvxBaseViewModel. This would also limit the breaking changes - developers would still need to add the parameters to their constructor but they wouldn't need to change their base class.